packages/ppx_compose/ppx_compose.0.0.3/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+name: "ppx_compose"
+authors: ["Petter A. Urkedal"]
+maintainer: "paurkedal@gmail.com"
+homepage: "https://github.com/paurkedal/ppx_compose/"
+bug-reports: "https://github.com/paurkedal/ppx_compose/issues"
+dev-repo: "https://github.com/paurkedal/ppx_compose.git"
+license: "LGPL-3 with OCaml linking exception"
+
+build: [["jbuilder" "build" "-p" name "-j" jobs]]
+build-test: [["jbuilder" "runtest" "-p" name "-j" jobs]]
+
+depends: [
+  "ocamlbuild" {build}
```

Comment:
I don't think ocamlbuild is necessary here since it uses jbuilder?
An explicit dependency on `jbuilder {>="1.0+beta7"}` is probably sufficient.